docs/Parallel-Learning-Guide.rst

```diff
@@ -62,6 +62,368 @@ Dask
 
 LightGBM's Python package supports distributed learning via `Dask`_. This integration is maintained by LightGBM's maintainers.
 
+Quick Dask Examples
+'''''''''''''''''''
+
+Theses examples show minimal code needed to train a LightGBM model using Dask. See the other sections after this for more details.
+
+**Classification**
+
+For multiclass classification, set ``centers=3`` below.
+
+.. code:: python
```

Comment:
@jameslamb I see this PR is WIP but I'd like to suggest one thing that can change the whole design of the PR.

WDYT about writing examples (even "quick") in a form of separate files in the [`examples/python-guide/Dask`](https://github.com/microsoft/LightGBM/tree/master/examples/python-guide) folder and have a reference for them in this doc? Going this way we will run all examples at our CI and be sure they are up-to-date and works fine.
https://github.com/microsoft/LightGBM/blob/87c37bf04f12cdf15498adfec6bf0baba5498ab2/.ci/test.sh#L221

Reply:
oh sure! I like that idea